internal/config/config.go

```diff
@@ -693,7 +714,7 @@ func validateBGPAdvPerPool(adv *BGPAdvertisement, pool *Pool) error {
 func ParseCommunity(c string) (uint32, error) {
 	fs := strings.Split(c, ":")
 	if len(fs) != 2 {
-		return 0, fmt.Errorf("invalid community string %q", c)
+		return 0, TransientError{fmt.Sprintf("invalid community string %q", c)}
```

Comment:
Why this is transient? If the parsing failed it will never pass.
The only case where it makes sense to have it transient is, when we have a string referencing a non existing community

Reply:
moved the TransientError out of the ParseComunnity function